api/API_VERSIONING.md

```diff
@@ -1,6 +1,6 @@
 # API versioning guidelines
 
-The Envoy project (and in the future [UDPA](https://github.com/cncf/udpa)) takes API stability and
+The Envoy project (and in the future [UDPA](https://github.com/cncf/xds)) takes API stability and
```

Comment:
`[XDS]`  ?

Reply:
`The Envoy project (and [xDS working group])`